app_dart/lib/src/service/scheduler.dart

```diff
@@ -349,8 +349,13 @@ class Scheduler {
       sha: pullRequest.head!.sha,
     );
     final CiYaml ciYaml = await getCiYaml(commit);
-    final Iterable<Target> presubmitTargets =
-        ciYaml.presubmitTargets.where((Target target) => target.value.scheduler == pb.SchedulerSystem.luci);
+    final Iterable<Target> presubmitTargets = ciYaml.presubmitTargets.where((Target target) =>
+        target.value.scheduler == pb.SchedulerSystem.luci || target.value.scheduler == pb.SchedulerSystem.cocoon);
```

Comment:
Do we have any builder set cocoon as the scheduler?

Reply:
Not yet, however as I work on implementing https://github.com/flutter/flutter/issues/92300 this will start to occur. ETA is end of month for a single builder running on the cocoon scheduler.